EXAMPLES.md

```diff
@@ -113,20 +113,30 @@ $ flict list
 
 # Create a policy report of the Europe project
 
-***This feature is currently unavailable***
+Create a compatibility, `fontconfig-report.json`, report for `fontconfig.json`:
+```
```

Comment:
Minor nitpick - between the text and a unformatted box there should always be a blank line.
Additionally pls specify the content of the code box by e.g. ```shell or similar

Reply:
I shell fix that